doc/src/manual/parallel-computing.md

```diff
@@ -1346,288 +1639,158 @@ Currently, sending a message between unconnected workers results in an error. Th
 as with the functionality and interface, should be considered experimental in nature and may change
 in future releases.
 
-## Multi-Threading (Experimental)
-
-In addition to tasks, remote calls, and remote references, Julia from `v0.5` forwards natively
-supports multi-threading. Note that this section is experimental and the interfaces may change
-in the future.
+## Noteworthy external packages
 
-### Setup
+Outside of Julia parallelism there are plenty of external packages that should be mentioned.
+For example [MPI.jl](https://github.com/JuliaParallel/MPI.jl) is a Julia wrapper for the `MPI` protocol, or
+[DistributedArrays.jl](https://github.com/JuliaParallel/Distributedarrays.jl), as presented in [Shared Arrays](@ref).
+A mention must be done to the Julia's GPU programming ecosystem, which includes :
 
-By default, Julia starts up with a single thread of execution. This can be verified by using the
-command [`Threads.nthreads()`](@ref):
+1. Low-level (C kernel) based operations [OpenCL.jl](https://github.com/JuliaGPU/OpenCL.jl) and [CUDAdrv.jl](https://github.com/JuliaGPU/CUDAdrv.jl) which are respectively a  OpenCL interface and a CUDA wrapper.
 
-```julia-repl
-julia> Threads.nthreads()
-1
-```
+2. High level (Julia Kernel) interfaces like [CUDAnative.jl](https://github.com/JuliaGPU/CUDAnative.jl) which is a Julia native CUDA implementation.
 
-The number of threads Julia starts up with is controlled by an environment variable called `JULIA_NUM_THREADS`.
-Now, let's start up Julia with 4 threads:
+3. High-level vendor specific abstractions [CuArray.jl](https://github.com/JuliaGPU/CuArrays.jl) and [CLArray.jl](https://github.com/JuliaGPU/CLArrays.jl)
```

Comment:
missing a word ("High-level vendor specific abstractions _like_")
It's also CuArrays.jl and CLArrays.jl